packages/monaco/src/split-editor.tsx

```diff
@@ -89,6 +91,50 @@ export default function SplitEditor({
   const monacoRef = useRef<typeof import('monaco-editor')>();
   const editorRef = useRef<monacoType.editor.IStandaloneCodeEditor>();
 
+  const [wrapperFocused, setWrapperFocused] = useState(false);
```

Comment:
quick question, why is it called wrapperFocused, i think the naming can be better for readability. just a thought.

Reply:
in this context the wrapper is just the wrapper div we have reference to with useRef, I think if we wanted to clarify I could rename to `isWrapperInnerFocused`

Reply:
since it's a boolean i would say it's a better and more expressive name